main/src/main/scala/sbt/Defaults.scala

```diff
@@ -200,7 +200,11 @@ object Defaults extends BuildCommon {
       pomPostProcess :== idFun,
       pomAllRepositories :== false,
       pomIncludeRepository :== Classpaths.defaultRepositoryFilter,
-      updateOptions := UpdateOptions(),
+      updateOptions := {
+        // This speeds up SNAPSHOT resolution for sbt itself
+        if (sbtPlugin.value) UpdateOptions().withLatestSnapshots(false)
```

Comment:
but `sbtPlugin` isn't only true in the metabuild

Reply:
Sure, but I think it's fine even for plugin development to have this flag on.

Reply:
would this not affect the resolution of sbt plugins in the metabuild?

Reply:
Yes. Since Bintray doesn't allow SNAPSHOT, the effect should be minimal there too.

Reply:
not for non-Bintray users..

I think some other flag for "I'm in the meta-build" should be introduced and used here, and avoiding the issue entirely.

Reply:
I think it's good to _not_ to make the distinction because the plugin authors would have closer emulation of the metabuild.

Reply:
> I think some other flag for "I'm in the meta-build" should be introduced and used here, and avoiding the issue entirely.